internal/plugin.go

```diff
@@ -700,6 +700,14 @@ func (p *Plugin) applyDefaults(unmarshaledConfig map[string]interface{}) {
 			policy.HubTemplateOptions.ServiceAccountName = p.PolicyDefaults.HubTemplateOptions.ServiceAccountName
 		}
 
+		if policy.CustomMessage.Compliant == "" {
+			policy.CustomMessage.Compliant = p.PolicyDefaults.CustomMessage.Compliant
```

Comment:
Should this use something similar to (or just reuse) the `isEvaluationIntervalSet()`?
```suggestion
			policy.CustomMessage.Compliant = p.PolicyDefaults.CustomMessage.Compliant
```

Reply:
Good point. That is more flexible.